backend/app/services/department_service.py

```
@@ -25,7 +25,7 @@ async def create_department(
     current_user: Users = Depends(get_current_user),
     db: AsyncSession = Depends(get_db),
     request_id: Optional[str] = None,
-    _: bool = Depends(require_permissions([Permission.CREATE_DEPARTMENT]))
+    _= Depends(require_permissions_dependency([Permission.CREATE_DEPARTMENT]))
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)